gotfparse/cmd/tftest/main.go

```diff
@@ -8,18 +8,41 @@ import (
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/cloud-custodian/tfparse/gotfparse/pkg/converter"
 )
 
 func main() {
-	if len(os.Args) < 2 || len(os.Args) > 2 {
+	if len(os.Args) < 2 {
```

Comment:
maybe it'd be worth to switch to use the `flag` package for argument parsing. Not a big deal though, given it's a test utility.

Reply:
Yeah, I considered switching to something but didn't want to get too crazy in this PR, I was already swapping to `slog` for logging.   I think we should file a ticket and then do a follow-up because it might make sense to swap to cobra as well (or maybe thats too much for a small util)